lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2967,6 +2974,23 @@ ParseResult FIRStmtParser::parseStaticRefExp(Value &result,
                  parseOptionalExpPostscript(result, false));
 }
 
+/// path ::= 'path(' StringLit ')'
+// NOLINTNEXTLINE(misc-no-recursion)
+ParseResult FIRStmtParser::parsePathExp(Value &result) {
+  auto startTok = consumeToken(FIRToken::lp_path);
+  locationProcessor.setLoc(startTok.getLoc());
+  StringRef target;
+  if (parseGetSpelling(target) ||
+      parseToken(FIRToken::string,
+                 "expected target string in path expression") ||
+      parseToken(FIRToken::r_paren, "expected ')' in path expression"))
+    return failure();
+  locationProcessor.setLoc(startTok.getLoc());
```

Comment:
Probably don't need to call `setLoc()` both here and above, don't think any of the methods between the two create operations or interact with that tracking?